src/test/java/org/testng/AssertTest.java

```diff
@@ -41,6 +41,13 @@ public void nullSetAssertEquals() {
     Assert.assertEquals(actual, expected);
   }
 
+  @Test
+  public void testSetAssertEquals() {

```

Comment:
I think you are using `assertEquals(Collection)` in the test.
But as `assertEquals(Set)` and `assertEquals(Collection)` are different, could you add a test for both methods?

Reply:
I have added test cases for list, set and collection having comparison with there equals implementation.